token_server/README.md

```diff
@@ -34,9 +34,59 @@ unless it is getting traffic.
 [Dashboard](https://console.cloud.google.com/appengine?folder=&organizationId=838088520005&project=mapping-crisis).
 You must be logged in as `gd-earthengine-user@givedirectly.org` to access it.
 
-## Run server on Amazon EC2
+### Set up new Google Cloud instance
+* Most of the necessary work needs to be done to set up the mapping page to work
```

Comment:
Having slight trouble parsing this first sentence

Reply:
Rephrased, pointing to future INITIAL_SETUP doc. better?